docs/concepts/macros/overview.md

```diff
@@ -0,0 +1,13 @@
+# Overview
+
+SQL is a static language. It does not have features like variables or control flow logic (if-then) that allow SQL commands to behave differently in different situations.
```

Comment:
I think I'd personally use the word _declarative_ instead of _static_ here, but I don't feel strongly about it. Maybe users will be able to understand the latter easier (?).